src/main/resources/lang/pylonbase/en.yml

```
@@ -141,6 +141,17 @@ item:
       <arrow> Slow, but can water lots of crops at once
       <arrow> <attr:Range:blocks>%range%</attr>
 
+  magic_dust:
+    name: "Magic dust"
+
+  beheading_sword:
+    name: "Beheading Sword"
+    lore: |-
+      <arrow> Gives you the heads of what you kill
+      <arrow> Causes players, creepers, piglins, ender dragons,
+        skeletons and zombies to drop their head with a 15% chance
+      <arrow> +2.5% chance for wither skeletons to drop their head
```

```suggestion
      <arrow> Gives you the heads of what you kill
      <arrow> Causes players, creepers, piglins, ender dragons, skeletons and zombies to drop their head with a 15% chance
      <arrow> +2.5% chance for wither skeletons to drop their head
```
This needs to be on one line, the line wrapper will automatically wrap it. All line breaks in this file will be interpreted as forced line breaks.